docs/apache-airflow/cli-and-env-variables-ref.rst

```diff
@@ -34,6 +34,12 @@ development and testing.
     :local:
     :depth: 2
 
+Providers that implement executors might contribute additional commands to the CLI. Here are the commands
+contributed by the community providers:
+
+* Celery Executor and related CLI commands: :doc:`apache-airflow-providers-celery:cli-ref`
+* Kubernetes Executor and related CLI commands: :doc:`apache-airflow-providers-cncf-kubernetes:cli-ref`
+
```

Comment:
Do we want this list to be auto generated/discovered like we do with Notifiers?

Reply:
I thought about it and I decided not to - maybe even forever.

Not for now - that's not as straightforward - also because the CLI commands can be technically added by different executors (LocalKubernetesExecutor has the same commands as KubernetesExecutor and CeleryKubernetesExecutor adds both Celery and Kubernetes one) - so we would have to somehow de-duplicate them - and it's not obvious how to fully automate that until we have more executors that are potentially extending those we already have.

Since those are only "executors" that can add the CLI commands, they will be added rarely enough so that we do not have to automatically discover those, I think.

But if we see the need in the future we can always add it.